extensions/data-workspace/src/dialogs/newProjectDialog.ts

```diff
@@ -0,0 +1,151 @@
+			if (!parentDirectoryExists) {
+				this.showErrorMessage(constants.ProjectParentDirectoryNotExistError(this.model.location));
+				return false;
+			}
+
+			// there shouldn't be an existing sub directory with the same name as the project in the selected location
+			const projectDirectoryExists = await directoryExist(path.join(this.model.location, this.model.name));
+			if (projectDirectoryExists) {
+				this.showErrorMessage(constants.ProjectDirectoryAlreadyExistError(this.model.name, this.model.location));
+				return false;
+			}
+
+			return true;
+		}
+		catch (err) {
+			this.showErrorMessage(err?.message ? err.message : err);
+			return false;
+		}
+	}
+
+	async onComplete(): Promise<void> {
+		try {
+			await this.workspaceService.createProject(this.model.name, vscode.Uri.file(this.model.location), this.model.projectTypeId);
+		}
+		catch (err) {
+			vscode.window.showErrorMessage(err?.message ? err.message : err);
+		}
+	}
+
+	protected async initialize(view: azdata.ModelView): Promise<void> {
+		const allProjectTypes = await this.workspaceService.getAllProjectTypes();
+		const projectTypeRadioCardGroup = view.modelBuilder.radioCardGroup().withProperties<azdata.RadioCardGroupComponentProperties>({
+			cards: allProjectTypes.map((projectType: IProjectType) => {
+				return <azdata.RadioCard>{
+					id: projectType.id,
+					label: projectType.displayName,
+					icon: projectType.icon,
+					descriptions: [
+						{
+							textValue: projectType.displayName,
+							textStyles: {
+								'font-size': '13px',
+								'font-weight': 'bold'
+							}
+						}, {
+							textValue: projectType.description
+						}
+					]
+				};
+			}),
+			iconHeight: '25px',
+			iconWidth: '25px',
+			cardWidth: '250px',
+			cardHeight: '130px',
+			ariaLabel: constants.ProjectTypeSelectorTitle,
+			width: '500px',
+			iconPosition: 'left',
+			selectedCardId: allProjectTypes.length > 0 ? allProjectTypes[0].id : undefined
+		}).component();
+
+		this.register(projectTypeRadioCardGroup.onSelectionChanged((e) => {
+			this.model.projectTypeId = e.cardId;
+		}));
+
+		const projectNameTextBox = view.modelBuilder.inputBox().withProperties<azdata.InputBoxProperties>({
+			ariaLabel: constants.ProjectNameTitle,
+			required: true,
+			width: constants.DefaultInputWidth
+		}).component();
+
+		this.register(projectNameTextBox.onTextChanged(() => {
+			this.model.name = projectNameTextBox.value!;
+		}));
+
+		const locationTextBox = view.modelBuilder.inputBox().withProperties<azdata.InputBoxProperties>({
+			ariaLabel: constants.ProjectLocationTitle,
+			required: true,
+			width: constants.DefaultInputWidth
+		}).component();
+		this.register(locationTextBox.onTextChanged(() => {
+			this.model.location = locationTextBox.value!;
+		}));
+
+		const browseFolderButton = view.modelBuilder.button().withProperties<azdata.ButtonProperties>({ label: constants.BrowseButtonText, width: constants.DefaultButtonWidth }).component();
```

Comment:
browse buttons should be folder icons to match the mockup

Reply:
I'll consistent with other places in ADS